### DISCUSSION TITLE
How to exclude templates using flag config file

### DISCUSSION THREAD

**Discussion by Bywalks** [projectdiscovery/nuclei#1655] [Q&A] [answered]
exclude-templates in config.yaml not work

For example:
my config:
severity: critical, high

exclude-templates: 
 - takeovers/github-takeover

Befor and after add exclude-templates,templates loaded for scan is 1131.

Nuclei version 2.6.2

**Comment by ehsandeep** [comment, marked as answer]
@Bywalks `exclude-templates` accept template path as input, so valid config example will be as follows:

```
exclude-templates:
  - takeovers/github-takeover.yaml
```
